meson.build

```diff
@@ -155,6 +155,9 @@ endif
 # Set configuration path
 configuration.set_quoted('CFGOPTDEF_CONFIG_PATH', get_option('configdir'), description: 'Configuration path')
 
+# Set FN_NO_RETURN macro
+configuration.set('FN_NO_RETURN', '__attribute__((__noreturn__))', description: 'Indicate that a function does not return')
```

Comment:
Note for the future, you can do configure time checks to see if this is supported via:

https://mesonbuild.com/Reference-manual_returned_compiler.html#compilerhas_function_attribute

Reply:
Thanks for this tip.

I went a bit through meson's reference, but didn't catch it.
I will definitely make use of that, when I start implementing the Windows port.

Thank you :).